packages/flutter/test/material/dropdown_test.dart

```diff
@@ -542,6 +542,34 @@ void main() {
     expect(buttonBox.size, equals(buttonBoxNullValue.size));
   });
 
+  testWidgets('Size of DropdownButton with no items', (WidgetTester tester) async {
+    final Key buttonKey = UniqueKey();
+    List<String> items;
+
+    Widget build() => buildFrame(buttonKey: buttonKey, items: items, onChanged: onChanged);
+
+    await tester.pumpWidget(build());
+    final RenderBox buttonBoxNullItems = tester.renderObject(find.byKey(buttonKey));
+    assert(buttonBoxNullItems.attached);
+
```

Comment:
please just leave one blank line between blocks of code, here and below

Reply:
Done